src/test/data/BigBed-test.js

```diff
@@ -147,6 +148,13 @@ describe('BigBed', function() {
       });
   });
 
+  it('file with 257 contigs', function () {
```

Comment:
Tests need a return type to pass flow. To fix this, change `function ()` to `function (): any`